cmd/snap-confine/snap-confine.c

```diff
@@ -86,33 +97,43 @@ int main(int argc, char **argv)
 #endif				// ifdef HAVE_SECCOMP
 
 	if (geteuid() == 0) {
-		const char *group_name = getenv("SNAP_NAME");
-		if (group_name == NULL) {
-			die("SNAP_NAME is not set");
+		if (!classic_confinement) {
+			/* 'classic confinement' is designed to run without the sandbox
```

Comment:
Can we move the comment above maybe? When I first saw the comment I assume it was meant for the following code.